modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -573,6 +596,7 @@ RenderableFieldlinesSequence::RenderableFieldlinesSequence(
 
     if (p.colorTableRanges.has_value()) {
         _colorTableRanges = *p.colorTableRanges;
+        _selectedColorRange = _colorTableRanges[_colorQuantityTemp];
```

Comment:
Is `_colorQuantityTemp` guaranteed to be >0 and < _colorTableRanges.size() here?  If not, it will lead to a hard crash